doc/guides/bluetooth/autopts/autopts-linux.rst

```diff
@@ -344,6 +349,16 @@ Server and client by default will run on localhost address. Run server:
    :width: 700
    :align: center
 
+.. note::
+
+    If the error "ImportError: No module named pywintypes" appeared after the fresh setup,
+    uninstall and install the pywin32 module:
+
+    .. code-block::
+
+        pip3 uninstall pywin32
+        pip3 install pywin32
```

Comment:
Is pip3/python3 the only Python that works with autopts? Otherwise I suggest using the more generic `pip` rather than `pip3`. 

```suggestion
        pip install --upgrade --force-reinstall pywin32
```

Reply:
With both pip and pip3 works fine, thanks!

Reply:
@mkasenberg Are you planning on updating this?

Reply:
Oops, missed that, sorry :)